OpenSSL/test/test_ssl.py

```diff
@@ -2998,6 +3008,15 @@ def test_bytearray_really_doesnt_overfill(self):
         """
         self._doesnt_overfill_test(bytearray)
 
+    def test_peek(self):
```

Comment:
oh and while we’re waiting, pyOpenSSL has sadly the gross code convention to leave two lines between methods on a class, so you’ll have to fix it here and in the read tests too.  I’m sorry, I really wish it hadn’t.